src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunReportPageTests.kt

```diff
@@ -92,28 +92,27 @@ class RunReportPageTests : SeleniumTest()
         assertThat(tab.findElement(By.tagName("p")).text).isEqualTo("Report logs coming soon!")
     }
 
-
     //TODO: This test case should be revisited as soon as test data is updated.
-    @Test
+
     @Ignore
+    @Test
     fun `can fill parameter textField`()
```

Comment:
This change seems to remove an existing test, but I guess the intention is to add a new one? If so then please ensure that the original is retained.